lib/Github/Api/Repository/Checks/CheckRuns.php

```diff
@@ -83,4 +83,16 @@ public function allForReference(string $username, string $repository, string $re
 
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/commits/'.rawurlencode($ref).'/check-runs', $params);
     }
+
+    /**
+     * @link https://docs.github.com/en/rest/reference/checks#rerequest-a-check-run
+     *
+     * @return array
+     */
+    public function rerequest(string $username, string $repository, int $checkRunId)
+    {
+        $this->acceptHeaderValue = 'application/vnd.github.antiope-preview+json';
```

Comment:
Is this accept header required for this api endpoint? I don't seem to find it in the api docs.

Reply:
You are right, it is the wrong header. I replaced it with the correct one that is [listed here](https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#rerequest-a-check-run)